exercises/markdown/canonical-data.json

```diff
@@ -76,6 +76,30 @@
         "markdown": "# Header!\n* __Bold Item__\n* _Italic Item_"
       },
       "expected": "<h1>Header!</h1><ul><li><strong>Bold Item</strong></li><li><em>Italic Item</em></li></ul>"
+    },
+    {
+      "description": "with markdown symbols in the header text that should not be interpreted",
+      "property": "parse",
+      "input": {
+        "markdown": "# This is a header with # and *** in the text"
+      },
+      "expected": "<h1>This is a header with # and *** in the text</h1>"
+    },
+    {
+      "description": "with markdown symbols in the list item text that should not be interpreted",
+      "property": "parse",
+      "input": {
+        "markdown": "* Item 1 with a # in the text\n* Item 2 with *** in the text"
+      },
+      "expected": "<ul><li>Item 1 with a # in the text</li><li>Item 2 with *** in the text</li></ul>"
+    },
+    {
+      "description": "with markdown symbols in the paragraph text that should not be interpreted",
+      "property": "parse",
+      "input": {
+        "markdown": "This is a paragraph with # and *** in the text"
+      },
+      "expected": "<p>This is a paragraph with # and *** in the text</p>"
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Some may criticize this as it is a change unrelated to the purpose of commit 70ffff5ee8d2cee582f240def607ab212ba36624.  This sort of thing happens pretty frequently, especially if using the GitHub editor.  I usually bring it to peoples attention but usually don't require it to be fixed.

Reply:
@rpottsoh Shoot... I'm going to close this PR and fix the version number and.. maybe whatever happened with the newline.

Reply:
Thanks for the heads up. I opened a new one, and didn't use the github editor.
https://github.com/exercism/problem-specifications/pull/1291